examples/guestbook/README.md

```diff
@@ -83,15 +158,17 @@ kind: ReplicationController
 metadata:
   name: redis-master
   labels:
-    name: redis-master
+    app: redis
+    tier: backend
+    role: master
 spec:
   replicas: 1
-  selector:
-    name: redis-master
   template:
     metadata:
       labels:
-        name: redis-master
+        app: redis
+        tier: backend
```

Comment:
I would put the tier labels above the app labels, everywhere, or at the bottom of the map, since they aren't truly necessary. It makes more sense for app and role to be back to back.